appveyor ci fails because asmap.raw.h is missing

Not sure how to fix this, so leaving a bug report

C:\projects\bitcoin\src\test\addrman_tests.cpp(9,10): fatal error C1083: Cannot open include file: 'test/data/asmap.raw.h': No such file or directory [C:\projects\bitcoin\build_msvc\test_bitcoin\test_bitcoin.vcxproj]

--- a/build_msvc/test_bitcoin/test_bitcoin.vcxproj
+++ b/build_msvc/test_bitcoin/test_bitcoin.vcxproj
@@ -61,7 +61,9 @@
     </PropertyGroup>
     <ItemGroup>
       <JsonTestFile Include="..\..\src\test\data\*.json" />
+      <RawTestFile Include="..\..\src\test\data\*.raw" />
     </ItemGroup>
+    <HeaderFromHexdump RawFilePath="%(RawTestFile.FullPath)" HeaderFilePath="%(RawTestFile.FullPath).h" SourceHeader="static unsigned const char %(RawTestFile.Filename)_raw[] = {" SourceFooter="};" />
     <HeaderFromHexdump RawFilePath="%(JsonTestFile.FullPath)" HeaderFilePath="%(JsonTestFile.FullPath).h" SourceHeader="namespace json_tests{ static unsigned const char %(JsonTestFile.Filename)[] = {" SourceFooter="};}" />
   </Target>
   <Import Project="$(VCTargetsPath)\Microsoft.Cpp.props" />

